index.d.ts

```diff
@@ -865,6 +874,10 @@ declare module 'plivo-browser-sdk/media/audioDevice' {
         * Check the input & output audio device for windows machine such that both belong to same groupid
         */
     export const setAudioDeviceForForWindows: (devices: any, lastConnectedMicDevice: any, lastConnectedSpeakerDevice: any) => void;
```

Comment:
typo in the name of the function double "For"

Reply:
Acknowledged. I'll fix it